notebooks/curation_api/python/create_dataset_from_local_file.ipynb

```diff
@@ -17,9 +17,9 @@
     "\n",
     "In order to use this script, you must have a Curation API key (obtained from upper-righthand dropdown in the CZ CELLxGENE Discover data portal after logging in).\n",
     "\n",
-    "_For **new** Datasets_: You must separately create a Dataset (not shown in this notebook). Then, use the returned `dataset_id` as the suffix (append to the `UploadKeyPrefix` returned from the `/s3-upload-credentials` endpoint) of the S3 upload key. See code below, or read more detailed instructions about how to submit Datasets via S3 upload in [the description for the credentials endpoint](https://api.cellxgene.cziscience.com/curation/ui/#/collection/backend.corpora.lambdas.api.v1.curation.collections.collection_id.datasets.upload_s3.get).\n",
+    "_For **new** Datasets_: You must separately create a Dataset (not shown in this notebook). Then, use the returned Dataset `id` as the suffix (append to the `UploadKeyPrefix` returned from the `/s3-upload-credentials` endpoint) of the S3 upload key. See code below, or read more detailed instructions about how to submit Datasets via S3 upload in [the description for the credentials endpoint](https://api.cellxgene.cziscience.com/curation/ui/#/collection/backend.corpora.lambdas.api.v1.curation.collections.collection_id.datasets.upload_s3.get).\n",
     "\n",
-    "_For **replacing/updating** existing Datasets_: Uploads to a `dataset_id` for which there already exists a Dataset in the given Collection will result in the existing Dataset being replaced by a new Dataset created from the datafile that you are uploading.\n",
+    "_For **replacing/updating** existing Datasets_: Uploads to a Dataset id for which there already exists a Dataset in the given Collection will result in the existing Dataset being replaced by a new Dataset created from the datafile that you are uploading.\n",
```

Comment:
Intentionally making this "Dataset id" and not "Dataset `id`" because it is not referring to the eponymous attribute on the response structure as it is above.